src/core/Akka/Serialization/Serializer.cs

```diff
@@ -121,7 +121,17 @@ public static int GetSerializerIdentifierFromConfig(Type type, ExtendedActorSyst
         {
```

Comment:
So ise the serializer.identifier obsolete and should be replaced with this new way of handling things?
It seems conflicting to have both? or maybe I'm just not understanding the difference :-)